config/database.yml

```diff
@@ -0,0 +1,67 @@
+# PostgreSQL. Versions 8.2 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On OS X with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On OS X with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem 'pg'
+#
+default: &default
+ adapter: postgresql
+ encoding: unicode
+# For details on connection pooling, see rails configuration guide
+# http://guides.rubyonrails.org/configuring.html#database-pooling
+ pool: 5
+ host: localhost
+
+development:
+ <<: *default
+ database: <%= ENV['CAB_DEV'] %>
+ username: <%= ENV['DB_USERNAME'] %>
+ password: <%= ENV['DB_PASSWORD'] %>
+ port: <%= ENV['POSTGRES_PORT'] %>
+
+
+# The specified database role being used to connect to postgres.
+# To create additional roles in postgres see `$ createuser --help`.
+# When left blank, postgres will use the default role. This is
+# the same name as the operating system user that initialized the database.
+#username: bullet_api
+
+# The password associated with the postgres role (username).
+#password:
+
+# Connect on a TCP socket. Omitted by default since the client uses a
+# domain socket that doesn't need configuration. Windows does not have
+# domain sockets, so uncomment these lines.
+#host: localhost
+
+# The TCP port the server listens on. Defaults to 5432.
+# If your server runs on a different port number, change accordingly.
+#port: 5432
+
+# Schema search path. The server defaults to $user,public
+#schema_search_path: myapp,sharedapp,public
+
+# Minimum log levels, in increasing order:
+#   debug5, debug4, debug3, debug2, debug1,
+#   log, notice, warning, error, fatal, and panic
+# Defaults to warning.
+#min_messages: notice
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+ <<: *default
+ database: <%= ENV['CAB_TEST'] %>
+ username: <%= ENV['DB_USERNAME'] %>
+ password: <%= ENV['DB_PASSWORD'] %>
```

Comment:
@akanshak984 Don't we add staging and production configuration?

Reply:
Yeah, I will update it!